src/redux/sagas/messages/index.ts

```diff
@@ -1,6 +1,6 @@
 import { nanoid } from 'nanoid';
 import { call, put, race, take } from 'redux-saga/effects';
-
+import { providers } from 'ethers';
 import { putError } from '../utils';
 import { ActionTypes } from '../../actionTypes';
 import { AllActions } from '../../types/actions';
```

Comment:
The import order looks a little broken here

Reply:
Other than not being a space between external and local import, I don't see a issue with this.

- first you have external imports
- then local utils
- then local types
- then different local types